### DISCUSSION TITLE
Multihost, wie Host entfernen im Docker Container

### DISCUSSION THREAD

**Discussion by tops4u** [buanet/ioBroker.docker#492] [Questions & Answers / Fragen & Antworten] [answered]
Ich hatte 2 IOBroker Hosts. 1. im Docker + 2. auf einem ODroid Gerät. Letzteres ist letzthin gestorben. Nun wollte ich diesen Host aus der Konfig entfernen, jedoch geht das scheinbar nur dann, wenn IOBroker gestoppt ist was aber im Docker Kontext nicht geht... 

Wie entferne ich einen 2. Host aus einer Docker Installation?

**Comment by buanet** [comment, marked as answer]
> wenn IOBroker gestoppt ist was aber im Docker Kontext nicht geht...

Doch, das geht.

https://docs.buanet.de/de/iobroker-docker-image/docs/#benutze-maintenance-script

Ich habe das Image so gebaut, dass auch ein stoppen des ioBroker möglich ist. 

MfG, 
André

**Comment by tops4u** [comment]
Super das hat geklappt, besten dank. Vielleicht wäre ein Hinweis darauf noch toll wenn man im Container >iobroker stop machen möchte.
